Microsoft/vscode-docs-authoring
# Jamarw/linting unit tests

Initial check-in for custom rule unit testing.  I'd also like to centralize some strings but I'll that in another PR so this one doesn't get too muddled.

**Next steps**

1. Centralize/update strings.
2. Update readme and dev guide.

**Description of changes**

1. Added unit tests.
2. Added test content.
3. Added markdownlint and jest as dependencies.
4. Updated moniker, syntax and zone rules to align with unit tests input.

**Testing**

1. Pull jamarw/linting-unit-tests and run npm install.
2. Open docs-markdown in a terminal window (VS Code, PS, command line, etc.).
3. Run **npm run test-rules** to launch moniker, zone and syntax tests.
4. All tests should pass.
